File: Programming-languages-and-laboratory/Python/Exercises/40_compito_python_20220620/Furgone.py
from Veicolo import Veicolo


class Furgone(Veicolo):

    def __init__(self, codice, marca,  modello,  targa,costo, posti,categoria, carico):
        super().__init__(codice, marca,  modello,  targa,costo)

        self._posti = posti
        self._categoria = categoria
        self._carico = carico

    def __str__(self):
        return "commerciale\t"+super().__str__() + "-\t-\t-\t"  +str(self._categoria)+"\t" +str(self._posti) +"\t"+str(self._carico)

File: II - Linguaggi di programmazione e laboratorio/Esame/Java/40_compito_java_2020_06_22/Furgone.java
import java.util.*;

public class Furgone extends Veicolo {

    private int posti;
    private String categoria;
    private boolean carico;

    public Furgone(int codice, String targa, String modello,
                   String marca, double costo, int posti, String categoria, boolean carico) {
        super(codice, targa, modello,
                marca, costo);
        this.posti = posti;
        this.categoria = categoria;
        this.carico = carico;
    }

    public String toString() {
        return "commerciale\t" + super.toString() + "\t-\t-\t" + posti + "\t" + categoria + "\t" + carico;
    }

}